website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -39,7 +39,10 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the Resource Group where the Elasticsearch resource should exist. Changing this forces a new Elasticsearch to be created.
 
-* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
+* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.\
+Please note that the SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.\
+Ex: If the plan ID is "planXYZ" and term is "Yearly", the SKU will be "planXYZ_Yearly".\
+You may find your eligible plans [here](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/elastic.ec-azure-pp/selectionMode~/false/resourceGroupId//resourceGroupLocation//dontDiscardJourney~/false/selectedMenuId/home/launchingContext~/%7B%22galleryItemId%22%3A%22elastic.ec-azure-ppess-consumption-2024%22%2C%22source%22%3A%5B%22GalleryFeaturedMenuItemPart%22%2C%22VirtualizedTileDetails%22%5D%2C%22menuItemId%22%3A%22home%22%2C%22subMenuItemId%22%3A%22Search%20results%22%2C%22telemetryId%22%3A%2262f8ce76-e5e4-4983-9d3e-5c608a0b2bff%22%7D/searchTelemetryId/cca0a8d3-f232-4156-948f-701a5d74a729) or in the online documentation [here](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/elastic.ec-azure-pp?tab=PlansAndPrice) for more details or in case of any issues with the SKU.
```

Comment:
Is there a link to the document instead of the Portal? Also, I didn't find the list of eligible plan+term from the both links, except a drop-down list like:

![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/7970698/45053f0b-d7a4-4725-876f-ba4b71819e60)

From which I have no idea about the actual plan name used.

Reply:
The Term (Ex: 1-Month means "Monthly" as given in the Example) and Plan details are present at Plans+Pricing and Usage Information + Support Tabs
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/2b11b772-3072-4f44-897c-8f20b7458470)
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/d1a4cf49-6bf6-4cd9-9326-4cc6839cc37e)


Unfortunately these are not elaborated in the other online doc link that I have pasted in the same comment (second link); however the Customers could still refer to it for other details.

Reply:
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/1acdc875-3012-4e9b-bcab-082e118c4143)